vagrant-service-manager.spec

```diff
@@ -69,6 +69,7 @@ popd
 %{vagrant_plugin_libdir}
 %exclude %{vagrant_plugin_cache}
 %{vagrant_plugin_spec}
+%{vagrant_plugin_instdir}/plugins/guests/
```

Comment:
I am not sure if we have files in %{vagrant_plugin_dir}/* related to plugins/guests . The question is do we really care about ruby code with respect to plugins/guests in the RPM. IMO if we have some files in %{vagrant_plugin_dir}/* with respect to plugins/guests, we should remove them from the RPM.